test/i18n.js

```diff
@@ -90,6 +90,174 @@ describe('static i18n', function() {
     const browserData = apos.i18n.getBrowserData(apos.task.getReq());
     assert.strictEqual(browserData.i18n.en.custom.customTestOne, 'Custom Test One From Subtype');
   });
+
+  it('should not offer adminLocale by default', function () {
+    assert.deepStrictEqual(apos.i18n.adminLocales, []);
+    assert.strictEqual(apos.i18n.defaultAdminLocale, null);
+    assert.strictEqual(
+      apos.user.schema.some((field) => field.name === 'adminLocale'),
+      false
+    );
+    const browserData = apos.i18n.getBrowserData(apos.task.getReq());
+    assert.strictEqual(browserData.locale, 'en');
+    assert.strictEqual(browserData.adminLocale, 'en');
+  });
+
+  it('should add user.adminLocale when configured', async function () {
+    await t.destroy(apos);
+    apos = await t.create({
+      root: module,
+      modules: {
+        '@apostrophecms/i18n': {
+          options: {
+            locales: {
+              en: {},
+              fr: {
+                prefix: '/fr'
+              }
+            },
+            adminLocales: [
```

Comment:
Hmm, I guess label and value separately here is OK since these might not be content locales at all. 👍